platforms/documentation/docs/build.gradle.kts

```diff
@@ -630,6 +630,8 @@ tasks.named<Test>("docsTest") {
     systemProperties.clear()
 
     filter {
+        // We don't maintain Java 7 on our CI infrastructure anymore
+        excludeTestsMatching("*java7CrossCompilation.sample")
```

Comment:
So this is pretty much untested now? Can we at least run `help` on it to make sure the build scripts compile?

Or can we remove the sample altogether? Or change it to cross compile to Java 8?

Reply:
> Can we at least run help on it to make sure the build scripts compile?

That is the minimum we can do